assets/css/siteWide/style.css

```diff
@@ -1,17 +1,15 @@
 body {
-    background-color: white !important;
```

Comment:
I am not sure we actually need `!important` here, but we can think about this next week.

Reply:
Yep I agree, Was messing around with something and think I forgot to remove it...